### ISSUE TITLE
date field value can't save when no JS

### ISSUE THREAD

**Issue by ivanteoh** [plomino/Plomino#668]
We got a requirement where Plomino form need to work without JavaScript. Currently date field not working when no JS. Any feedbacks on which directions should go:
* Update the current date widget to support when no JS is enable. Could it break backward compatible
* Add new data widget as custom Plomino field type plugin.

**Comment by ebrehault** [comment]
I think we need a new date widget (named "Static html"), but there is no reason to put it in a custom plugin, just push it in the core.

**Comment by djay** [comment]
@ebrehault what is the reason its submitted currently as a single field? Seems that the separated fields are all there but not processed. If we processed those in teh case of no JS, is there an issue we haven't considered with that?

**Comment by djay** [comment]
Looks like we can change the names for the fields to use :tuple, :list, or :record converts right @ivanteoh. This will get around the fact that processInput only takes a single value. Would this work you think @ebrehault ?

**Comment by ebrehault** [comment]
yes it should work, I do  not know why it is splitted in separated fields, that's the way the Archetypes widget was implemented.
